enzyme/Enzyme/EnzymeLogic.cpp

```diff
@@ -3190,8 +3190,8 @@ Function *EnzymeLogic::CreatePrimalAndGradient(
   bool diffeReturnArg = key.retType == DIFFE_TYPE::OUT_DIFF;
 
   DiffeGradientUtils *gutils = DiffeGradientUtils::CreateFromClone(
-      *this, key.mode, key.todiff, TLI, TA, key.retType, diffeReturnArg,
-      key.constant_args, retVal, key.additionalType, omp);
+      *this, key.mode, /* width */ 1, key.todiff, TLI, TA, key.retType,
```

Comment:
This should also have the width passed through.

Reply:
done